contrib/platform/src/com/sun/jna/platform/win32/Kernel32.java

```diff
@@ -1162,7 +1163,7 @@ Pointer MapViewOfFile(HANDLE hFileMappingObject, int dwDesiredAccess,
      *         the function fails, the return value is zero. To get extended
      *         error information, call GetLastError.
      */
-    public boolean GetComputerName(char[] buffer, IntByReference lpnSize);
+    public boolean GetComputerName(NativeWin32String buffer, IntByReference lpnSize);
```

Comment:
See remarks for User32.GetWindowText,  Same for most of the `char[]` in this file.